tests/datasets/test_malaria.py

```diff
@@ -0,0 +1,7 @@
+import keras_rcnn.datasets.malaria
+
+def test_load_data():
+    training, test = keras_rcnn.datasets.malaria.load_data()
+
+    assert len(training) == 1208
+    assert len(test) == 120
```

Comment:
I’d also validate whether the value of a filename (e.g. from the first dictionary of training and test) exists.